layout/templates/futureWarming.mustache

```diff
@@ -5,10 +5,16 @@
 </div>
 
 <div id="futureWarming" class="center">
+<ul class='list'>
```

Comment:
@lawinslow see this and caption text for a couple changes I made to your section. merge if 👍 , as the rest of this includes minor changes.